### PR TITLE
Fixes #11259: Move hammer packages to SCL

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
This is so that our hammer packages will continue to work after this update https://github.com/theforeman/foreman-packaging/commit/d2718ada24ac5e77d2e98939ec4c39f5ba5fb15e and thus also unstick the pipeline.

File: rubygem-hammer_cli_csv/rubygem-hammer_cli_csv.spec [hunk 1]
```diff
@@ -1,49 +1,60 @@
...
-URL: https://github.com/Katello/hammer-cli-csv
+URL:     https://github.com/Katello/hammer-cli-csv
 Source0: https://rubygems.org/downloads/%{gem_name}-%{version}.gem
 
-%if 0%{?rhel} == 6
-Requires: ruby(abi)
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+%if 0%{?scl:1}
+Obsoletes: rubygem-%{gem_name} < 0.0.16-1
```
Comment (jlsherrill) [inline rubygem-hammer_cli_csv/rubygem-hammer_cli_csv.spec]: 0.0.16-1 ?

File: rubygem-hammer_cli_gutterball/rubygem-hammer_cli_gutterball.spec [hunk 2]
```diff
@@ -1,55 +1,60 @@
...
-URL: http://github.com/Katello/hammer-cli-gutterball
+URL:     http://github.com/Katello/hammer-cli-gutterball
 Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
 
-%if 0%{?rhel} == 6
-Requires: ruby(abi)
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+%if 0%{?scl:1}
+Obsoletes: rubygem-%{gem_name} < 0.0.16-1
```
Comment (jlsherrill) [inline rubygem-hammer_cli_gutterball/rubygem-hammer_cli_gutterball.spec]: 0.0.16-1 ?

File: rubygem-hammer_cli_import/rubygem-hammer_cli_import.spec [hunk 3]
```diff
@@ -1,71 +1,84 @@
...
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
 
-Group:      Development/Languages
-License:    GPLv3
-URL:        https://github.com/Katello/hammer-cli-import
-Source0:    http://rubygems.org/gems/%{gem_name}-%{version}.gem
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+%if 0%{?scl:1}
+Obsoletes: rubygem-%{gem_name} < 0.0.16-1
```
Comment (jlsherrill) [inline rubygem-hammer_cli_import/rubygem-hammer_cli_import.spec]: 0.0.16-1 ?

File: rubygem-hammer_cli_csv/rubygem-hammer_cli_csv.spec (hunk 1 above)
Comment (ehelms) [inline rubygem-hammer_cli_csv/rubygem-hammer_cli_csv.spec]: I don't understand the question?

Comment (ehelms) [inline rubygem-hammer_cli_csv/rubygem-hammer_cli_csv.spec]: Ohhh, copy paste fail!

**Comment by ehelms** [comment]
@jlsherrill updated the obsoletes to the proper ones and also realized I wasn't setting it to the next release. For example, if the current package is 0.0.16-1, the obsoletes is set to `< 0.0.16-2` which should be the next release with SCL.

**Comment by jlsherrill** [comment]
APJ

**Pull request merged**